portfolio/src/main/webapp/designs.html

```diff
@@ -7,71 +7,83 @@
 		<link rel="stylesheet" href="style.css" />
 	</head>
 	
-	<body class="is-preload">
-
+	<body>
 		<!-- Wrapper -->
 		<div id="wrapper">
 
 			<!-- Main -->
-                	<section id="main">
+            <section id="design-main">
+
+                <ul class="navbar">
+                    <li><a href="https://futuresheeos.github.io/">SheEO</a></li>
+                    <li><a class="active" href="designs.html">Design</a></li>
+                    <li><a href="projects.html">Projects</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="index.html">Home</a></li>
+                </ul>
+
 				<header>
-					<h3 class="subtitle">Design</h4>
+					<h3 class="subtitle"><strong>Design</strong></h4>
```

Comment:
I think the best practice is to avoid html tags like strong in favor of using a span and then styling it in the css: https://html.com/tags/strong/